x/evm/keeper/msg_server.go

```diff
@@ -1458,13 +1458,19 @@ func (s msgServer) SignCommands(c context.Context, req *types.SignCommandsReques
 	}
 
 	commandList := types.CommandIDsToStrings(commandBatch.GetCommandIDs())
-	for _, commandID := range commandList {
-		s.Logger(ctx).Debug("signing command batch", "commandBatchID", batchedCommandsIDHex, "commandID", commandID)
+	for _, commandIDHex := range commandList {
+		commandID, err := strconv.ParseUint(commandIDHex, 16, 64)
```

Comment:
@jcs47 I made a mistake here -  ParseUint expects bit size not byte size.

Reply:
Why convert to uint? not all command id origins from transfer id

Reply:
Same question. Also, these command IDs are of `bytes32` in the contracts which means it would flow for `ParseUint`.

Reply:
Same question. Also, these command IDs are of `bytes32` in the contracts which means it would overflow for `ParseUint`.

Reply:
This was a naive quick fix to try and address the mapping issue on call 😢. Reverting.